### ISSUE TITLE
{Linked:Bug9980704; Linked:Bug1560766} [Adaptive Cards - Designer]: In 400% zoom mode, 'Report below' and 'local emergency' text is getting cropped.  

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8296]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2 (OS Build 22590.1010)
Browser Version: Version 111.0.1660.7 (Official build) dev (64-bit)

**Pre-Requisite:**
Step 1 : Enable the Display resolution to 1280*1024.
Step 2 : Enable Scale and layout to 100%.
Step 3 : Enable the browser setting to 400% zoom mode and reload the page.

**Repro Steps:**
1. Open the above URL is edge dev browser.
2. Navigate to the 'Card payload Editor' and activate it.
3. Paste the attached JSON.
4. Verify the text is getting cropped or not?

**Actual Result:**
In 400% zoom mode, 'Report below' and 'Emergency services' text is getting cropped.

**Expected Result:**
In 400% zoom mode, 'Report below' and 'Emergency services' text should be visible properly.

**User Impact:**
Users who rely on the large text for navigation through the page are getting impacted, as after resizing the page to 400% zoom mode, "Report it Now" card text getting truncated.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B2ECDA461-6B98-45C8-8FB1-A9399A72DA17%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&DefaultItemOpen=1

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![MAS1 4 10 - Text is getting cropped at 400%](https://user-images.githubusercontent.com/95913748/217727852-d058f11e-901e-45b5-bc45-e6ab7e9f8ed7.png)

### Card JSON

```JSON
{
	"schema": "http=//adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.2",
	"body": [
		{
			"id": "",
			"isVisible": true,
			"separator": false,
			"spacing": "default",
			"type": "Container",
			"items": [
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "Medium",
					"type": "Container",
					"items": [
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "TextBlock",
							"textStyle": "default",
							"wrap": true,
							"text": "Do you have a security concern?",
							"size": "medium",
							"weight": "bolder"
						},
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "TextBlock",
							"textStyle": "default",
							"wrap": true,
							"text": "Chaitanya, thanks for letting us know. File a report below.",
							"size": "small",
							"weight": "bolder"
						},
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "TextBlock",
							"textStyle": "default",
							"wrap": true,
							"text": "For life threatening emergencies, please call local emergency services.",
							"size": "small",
							"weight": "default",
							"color": "attention"
						}
					],
					"verticalContentAlignment": "center",
					"bleed": false
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "default",
					"width": "250px",
					"type": "Image",
					"height": "150px",
					"horizontalAlignment": "center",
					"size": "auto",
					"style": "default",
					"url": "https://microsoft.sharepoint.com/sites/catalog/ClientSideAssets/12345678-9f17-4b50-8e1f-ea3145d8e95f/HomeScreenImage_6bc1d584f530be38f7c5296f2a94d30d.svg",
					"altText": "Home Image"
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "default",
					"type": "ActionSet",
					"actions": [
						{
							"style": "default",
							"id": "ReportItNowPage",
							"title": "File a report",
							"altText": "File a report",
							"type": "Action.OpenUrl",
							"url": "https://aka.ms/dev-reportitnow"
						}
					]
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "default",
					"type": "Container",
					"items": [
						{
							"id": "",
							"isVisible": true,
							"separator": false,
							"spacing": "default",
							"type": "TextBlock",
							"textStyle": "default",
							"wrap": true,
							"text": "Common Security Problems",
							"size": "medium",
							"weight": "bolder"
						}
					],
					"verticalContentAlignment": "center",
					"bleed": false
				},
				{
					"id": "",
					"isVisible": true,
					"separator": false,
					"spacing": "Medium",
					"type": "Container",
					"items": [
						{
							"id": "",
							"isVisible": true,
							"separator": true,
							"spacing": "default",
							"type": "Container",
							"items": [
								{
									"id": "",
									"isVisible": true,
									"separator": false,
									"spacing": "default",
									"type": "ColumnSet",
									"bleed": false,
									"columns": [
										{
											"type": "Column",
											"separator": false,
											"spacing": "default",
											"verticalContentAlignment": "center",
											"width": "stretch",
											"isVisible": true,
											"items": [
												{
													"id": "",
													"isVisible": true,
													"separator": false,
													"spacing": "default",
													"type": "TextBlock",
													"textStyle": "default",
													"wrap": true,
													"text": "Misplaced Badge?",
													"color": "dark",
													"size": "medium",
													"weight": "default"
												}
											]
										},
										{
											"type": "Column",
											"separator": false,
											"spacing": "default",
											"verticalContentAlignment": "center",
											"width": "auto",
											"isVisible": true,
											"items": [
												{
													"id": "",
													"isVisible": true,
													"separator": false,
													"spacing": "default",
													"width": "20px",
													"type": "Image",
													"height": "20px",
													"horizontalAlignment": "left",
													"size": "small",
													"style": "default",
													"url": "https://microsoft.sharepoint.com/sites/catalog/ClientSideAssets/12345678-9f17-4b50-8e1f-ea3145d8e95f/Chevron_e98408159aa3d2f02623abf4901fb147.png",
													"altText": "Report Misplaced Badge?"
												}
											]
										}
									]
								}
							],
							"verticalContentAlignment": "center",
							"bleed": false,
							"selectAction": {
								"style": "default",
								"id": "MisplacedBadgeRequest",
								"type": "Action.Submit",
								"associatedInputs": "auto",
								"data": { "id": "MisplacedBadgeRequest" }
							},
							"minHeight": "30px"
						},
						{
							"id": "",
							"isVisible": true,
							"separator": true,
							"spacing": "default",
							"type": "Container",
							"items": [
								{
									"id": "",
									"isVisible": true,
									"separator": false,
									"spacing": "default",
									"type": "ColumnSet",
									"bleed": false,
									"columns": [
										{
											"type": "Column",
											"separator": false,
											"spacing": "default",
											"verticalContentAlignment": "center",
											"width": "stretch",
											"isVisible": true,
											"items": [
												{
													"id": "",
													"isVisible": true,
													"separator": false,
													"spacing": "default",
													"type": "TextBlock",
													"textStyle": "default",
													"wrap": true,
													"text": "Lost SAW?",
													"color": "dark",
													"size": "medium",
													"weight": "default"
												}
											]
										},
										{
											"type": "Column",
											"separator": false,
											"spacing": "default",
											"verticalContentAlignment": "center",
											"width": "auto",
											"isVisible": true,
											"items": [
												{
													"id": "",
													"isVisible": true,
													"separator": false,
													"spacing": "default",
													"width": "20px",
													"type": "Image",
													"height": "20px",
													"horizontalAlignment": "left",
													"size": "small",
													"style": "default",
													"url": "https://microsoft.sharepoint.com/sites/catalog/ClientSideAssets/12345678-9f17-4b50-8e1f-ea3145d8e95f/Chevron_e98408159aa3d2f02623abf4901fb147.png",
													"altText": "Report Lost SAW?"
												}
											]
										}
									]
								}
							],
							"verticalContentAlignment": "center",
							"bleed": false,
							"selectAction": {
								"style": "default",
								"id": "LostSawRequest",
								"type": "Action.Submit",
								"associatedInputs": "auto",
								"data": { "id": "LostSawRequest" }
							},
							"minHeight": "30px"
						},
						{
							"id": "",
							"isVisible": true,
							"separator": true,
							"spacing": "default",
							"type": "Container",
							"items": [
								{
									"id": "",
									"isVisible": true,
									"separator": false,
									"spacing": "default",
									"type": "ColumnSet",
									"bleed": false,
									"columns": [
										{
											"type": "Column",
											"separator": false,
											"spacing": "default",
											"verticalContentAlignment": "center",
											"width": "stretch",
											"isVisible": true,
											"items": [
												{
													"id": "",
													"isVisible": true,
													"separator": false,
													"spacing": "default",
													"type": "TextBlock",
													"textStyle": "default",
													"wrap": true,
													"text": "Received Phishing Email?",
													"color": "dark",
													"size": "medium",
													"weight": "default"
												}
											]
										},
										{
											"type": "Column",
											"separator": false,
											"spacing": "default",
											"verticalContentAlignment": "center",
											"width": "auto",
											"isVisible": true,
											"items": [
												{
													"id": "",
													"isVisible": true,
													"separator": false,
													"spacing": "default",
													"width": "20px",
													"type": "Image",
													"height": "20px",
													"horizontalAlignment": "left",
													"size": "small",
													"style": "default",
													"url": "https://microsoft.sharepoint.com/sites/catalog/ClientSideAssets/12345678-9f17-4b50-8e1f-ea3145d8e95f/Chevron_e98408159aa3d2f02623abf4901fb147.png",
													"altText": "Report Received Phishing Email?"
												}
											]
										}
									]
								}
							],
							"verticalContentAlignment": "center",
							"bleed": false,
							"selectAction": {
								"style": "default",
								"id": "PhishingEmailRequest",
								"type": "Action.Submit",
								"associatedInputs": "auto",
								"data": { "id": "PhishingEmailRequest" }
							},
							"minHeight": "30px"
						}
					],
					"verticalContentAlignment": "center",
					"bleed": false
				}
			],
			"verticalContentAlignment": "center",
			"bleed": false
		}
	],
	"actions": [],
	"refresh": null
}
```

**Comment by Raisul123** [comment]
Dependent Bug ID https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/9980704

**Comment by jwoo-msft** [comment]
@anna-dingler 
I've assigned a11y task to you, and feel free to assign some of them to me.
Thanks!

**Comment by anna-dingler** [comment]
Hi @vagpt,

This bug was fixed with PR #8454, so I am closing this issue. Please let me know if you have any questions!

<img width="494" alt="image" src="https://user-images.githubusercontent.com/98650930/233221266-5de20fdf-950b-4bd5-8733-aa47d4ed680f.png">

**Comment by vagpt** [comment]
This issue is working fine and please refer to the below snippet for reference.

**Snippet:**

![image](https://user-images.githubusercontent.com/95913748/235113942-a73f42e9-299e-4f1d-a99c-987057616a99.png)
